dataactcore/utils/fileE.py

```diff
@@ -102,10 +151,17 @@ def suds_to_row(suds_obj):
 
 
 def retrieve_rows(duns_list):
-    """Soup-to-nuts creates a list of Row tuples from a set of DUNS
-    numbers."""
+    """ Soup-to-nuts creates a list of Row tuples from a set of DUNS numbers.
+
+        Args:
+            duns_list: A list of DUNS to search for
+    """
     if config_valid():
-        client = Client(CONFIG_BROKER['sam']['wsdl'])
+        try:
+            client = Client(CONFIG_BROKER['sam']['wsdl'])
```

Comment:
This line is copied in `dataactcore/utils/parentDuns.py`. Could you include this check in there too, or maybe refactor it out into a helper function which both can use?